groups/instance/iam.tf

```diff
@@ -62,6 +62,18 @@ data "aws_iam_policy_document" "kms_key" {
   }
 }
 
+data "aws_iam_policy_document" "access_ssm_parameters_policy_document" {
+  statement {
+    sid       = "AllowAccessToSsmParameters"
+    effect    = "Allow"
+    resources = ["arn:aws:ssm:${var.region}:${local.account_id}:parameter/${var.service}/${var.environment}/*"]
+    actions = [
+      "ssm:GetParametersByPath",
```

Comment:
I think you only need `ssm:GetParameter` as only getting a single parameter per each aws cli command - that worked ok for me on the prometheus changes: https://github.com/companieshouse/prometheus-terraform/blob/master/groups/prometheus/module-prometheus/iam-role.tf#L35